docs/content/usage/integrations/scoutsuite.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Scoutsuite"
+date: 2021-03-02T16:23:06+02:00
+draft: false
+description: "Scout Suite is an open-source cloud security-auditing tool."
+weight: 10
+---
+
+### About ScoutSuite
+
+<a href="https://github.com/nccgroup/ScoutSuite" target="_blank" >Scout Suite</a> is an open-source cloud security-auditing tool.
+It queries the cloud API to gather configuration data. Based on configuration
+data gathered, ScoutSuite shows security issues and risks present in your infrastructure.
+
+### Supported cloud providers
+
+Currently, the Infection Monkey only supports AWS.
```

Comment:
In the context of scoutsuite, this makes sense, but I wonder if someone may still find it a bit confusing. What about,

"Currently, the Infection Monkey can only use ScoutSuite to assess AWS environments."

Reply:
Reworded